de.metas.contracts/src/main/java/de/metas/contracts/inoutcandidate/SubscriptionShipmentScheduleHandler.java

```diff
@@ -99,14 +100,31 @@
 		documentLocationBL.setBPartnerAddress(documentLocation);
 
 		final I_C_Order order = create(term.getC_Order_Term(), I_C_Order.class);
-		newSched.setDeliveryRule(order.getDeliveryRule());
-		newSched.setDeliveryViaRule(order.getDeliveryViaRule());
+		if (order == null)
```

Comment:
i think it would be better to *alwasy* go with the term's values..the are set from the order when the term is created in `SubscriptionBL`, so they are there. and they might be changed in later terms..so the old order's rules might even be wrong...I'm not 100% sure if the old order is forwarded to reference, if a term is extended..even if it isn't we might change that in future